app/api/v2/__init__.py

```diff
@@ -0,0 +1,12 @@
+from flask import Blueprint
+from flask_restful import Api
+from .views.incidences_views import RedFlags, RedFlagsSpecific
+
+version_two = Blueprint("v2", __name__, url_prefix="/api/v2")
+API = Api(version_two)
+
+
+API.add_resource(RedFlags, '/incidences')
+API.add_resource(RedFlagsSpecific, '/incidences/<int:id>')
+API.add_resource(RedFlags, '/users')
+API.add_resource(RedFlagsSpecific, '/users/<int:id>')
```

Comment:
Lines 11 and 12 are the ones causing the issues with Blueprints. The resources `RedFlags` and `RedFlagsSpecific` are being repeated on `/users` routes. The effect of that is overriding the existing routes of `/incidences`. 

Get rid of the 2 lines and your blueprints should work.

Reply:
Resolved. Thank you.